### ISSUE TITLE
label widgets breaks on text with lines more than 950 ie. no text is rendered

### ISSUE THREAD

**Issue by peternasser99** [kivy/kivy#8213]
**Software Versions**
* Python:Python 3.9.4
* OS:windoes 10
* Kivy:2.1.0
* Kivy installation method: pip install kivy

**Describe the bug**
label widgets breaks on text with lines more than 950 ie. no text is rendered

**To Reproduce**
from kivy.app import App
from kivy.uix.label import Label

class MyApp(App):
    def build(self):
        # Create a string with 1000 lines, each 100 characters long
        text = ""
        for i in range(1000):
            text += "X" * 100 + "\n"
        
        # Create a label with the text
        label = Label(text=text)
        
        return label

if __name__ == '__main__':
    MyApp().run()

**Comment by misl6** [comment]
Hi @peternasser99 !

I suspect a buffer issue.
Can you please provide a full log (console output)?

**Comment by peternasser99** [comment]
The script runs as normal no errors or exceptions
but if you run the code above the label will be empty
(Either on my labtop or android phone s7 abit old )

**Comment by akshayaurora** [comment]
I guess we need a Long text display widget.
Label was originally only for displaying short text.

You can either try using TextInput with readonly set to True or similar approach.

**Comment by peternasser99** [comment]
further note 
its not a matter of text length but the number of new line characters \n 
```
        for i in range(1000):
            text += "X" * 100 + "\n"
is the same as 
        for i in range(1000):
            text += "X" * 1 + "\n"
```

**Comment by akshayaurora** [comment]
Yep,  texture gets big vertically even if there is no, text. `\n` signifies a empty line so a vertical height of line is added foe each `\n`

**Comment by akshayaurora** [comment]
I wonder why a simple textinput like so::

```
TextInput
    text: large_text
    readonly: True
```

Would not be good enough ?

**Comment by akshayaurora** [comment]
The difference with Label is textinput only renders one line of texture at a time and only combines /deals with a maximum number of lines visible on screen at a time.

Technically you can set properties like readonly,  show_handles, allow_copy etc to essentially get to a large text displaying.
Then use the new Scrolling behaviour introduced to allow for a scrolling large text display.

However we definitely should/need to make a simpler Widget that users can just use for this purpose available readily.

**Comment by akshayaurora** [comment]
This simple template seems to work.

```
<LargeLabel@TextInput>
	readonly: True
	on_focus: self.focus = False # avoids any interactions
	scroll_from_swipe: True

LargeLabel
    text: ''.join([f'Large Text{x}\n' for x in range(999)])
```

**Comment by peternasser99** [comment]
@akshayaurora 
`
This simple template seems to work.

<LargeLabel@TextInput>
	readonly: True
	on_focus: self.focus = False # avoids any interactions
	scroll_from_swipe: True

LargeLabel
    text: ''.join([f'Large Text{x}\n' for x in range(999)])
`
this actually works thanks 👍👍
